arbnode/dataposter/dbstorage/storage.go

```diff
@@ -61,7 +61,7 @@ func (s *Storage) Get(_ context.Context, index uint64) (*storage.QueuedTransacti
 	key := idxToKey(index)
 	value, err := s.db.Get(key)
 	if err != nil {
-		if errors.Is(err, leveldb.ErrNotFound) {
+		if isErrNotFound(err) {
```

Comment:
I don't think there's an `isErrNotFound` in scope here. Are you sure you didn't mean `dbutil.IsErrNotFound` ?

Reply:
Yes, you're right. The code has been moved recently. I cherry picked from our fork where the function is still in this file.

Reply:
https://github.com/OffchainLabs/nitro/pull/2532/commits/c227bab1812a003e9b0303912d5b8889cb31bc48